mkosi/__init__.py

```diff
@@ -2241,6 +2242,56 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        import portage
+    except ImportError:
+        die("you need portage module for Gentoo: https://gitweb.gentoo.org/proj/portage.git")
+
+    from _emerge.actions import load_emerge_config
+    from _emerge.actions import action_info
+    from _emerge.actions import action_sync
+    from _emerge.actions import action_build
+
+    portdir = "/var/db/repos/gentoo"
+    pkgdir = "/var/cache/binpkgs"
+    os.makedirs(portdir, exist_ok=1)
+    os.makedirs(pkgdir, exist_ok=1)
+
+    # TODO: add hooks for setting profile in args.gentoo_profile
+    os.symlink(
+        os.path.join(portdir, "profiles/default/linux/amd64/17.1/no-multilib"),
```

Comment:
What is the `17.1` here? That number seems somewhat magical to me.

Reply:
```17.1``` is just the latest profile version; there is 17.0 but that's about it. gentoo folks only support 17.1 btw.

but more important bit there is the ARCH (amd64) i think. this would matter for IoT et al.

in general profile in something left to users to choose; see here for more:
 https://wiki.gentoo.org/wiki/Profile_(Portage) 
i hesitated exposing this via, say, ```---gentoo-profile```/```GentooProfile```; it's internal to gentoo and i don't know if there is an equivalence for it in other distros or if if there is one whether ```mkosi``` supports such thing.

i will put together something for now.

Reply:
Sorry for replying late. I don't think there is a need to add distro specific flags. We've managed without so far. If the profiles are something like the release versions for other profiles, than it would make sense to reuse those flags.